src/CRM/CivixBundle/Resources/views/Code/entity-bao.php.php

```diff
@@ -1,10 +1,14 @@
 <?php
 echo "<?php\n";
+?>
+
+declare(strict_types = 1);
+<?php
```

Comment:
This formatting seems unnecessarily messy. Why not keep the php code together instead of splitting it? E.g. the `echo` and the `$_namespace` can all stay within the same code block so we don't need so many extra opening and closing php tags.